examples/gno.land/p/demo/ownable/ownable.gno

```diff
@@ -12,14 +10,16 @@ type Ownable struct {
 	owner std.Address
 }
 
-func New() *Ownable {
+func NewOwnable() *Ownable {
```

Comment:
I prefer `ownable.New()` over `ownable.NewOwnable()`. Some linters recommend using the shorter `ownable.New()` function.

Reply:
The issue that I was having is that when you embed the ownable object into Authorizeable, you have two declarations of `New()` on the struct. I suggest we keep ownable.New, and have `authorizable.NewAuthorizable()` WDYT?

Reply:
[9a9e215](https://github.com/gnolang/gno/pull/2372/commits/9a9e215feb07d596c0d5f85e6a91f0ab97728ef3)